core/integration/client_tests.go

```diff
@@ -93,6 +96,34 @@ func WithOutgoingFakeAuth(ctx context.Context, userID string) context.Context {
 	return metadata.NewOutgoingContext(ctx, metadata.New(md))
 }
 
+// SaveTestVectors generates test vectors for interoprability testing.
+func SaveTestVectors(env *Env, resps []testdata.GetEntryResponseVector) error {
+	// Output all key material needed to verify the test vectors.
+	domainFile := "../../core/testdata/domain.json"
```

Comment:
This seems to go all the way out of the core directory and back in, when this file is already under core?  Couldn't this just be "../testdata/domain.json"?

Reply:
The integration tests are launched from `imp/integration` and go tests are given the working directory of the test, hence the relative redirect.